src/cpp/session/modules/SessionPackages.R

```diff
@@ -1346,6 +1384,55 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
    
 })
 
+# Formats a person object in a way suitable for the Authors@R
+# section of a DESCRIPTION file.
+.rs.addFunction("formatPerson", function(person)
+{
+   personFields <- unclass(person)
+   if (length(personFields) == 1L)
+   {
+      formattedPerson <- .rs.formatPersonImpl(personFields[[1L]])
+      return(gsub("\n", "\n  ", formattedPerson))
+   }
+   
+   formattedPersons <- paste(
+      .rs.mapChr(personFields, .rs.formatPersonImpl, indent = "    "),
+      collapse = ",\n"
+   )
+   
+   paste(c("c(", formattedPersons, "  )"), collapse = "\n")
+})
+
+.rs.addFunction("formatPersonImpl", function(fields, indent = identity)
```

Comment:
It's been a long time since I last touched this in usethis, so my memory is faint. But I seem to have used an existing format method for this? Could you do the same?

https://github.com/r-lib/usethis/blob/a225a21f3e336394051584411efb4d351aa9d78a/R/description.R#L113-L114

Reply:
I thought about this, but I didn't like the default formatting produced by the base R format method. 😅